packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/api/resetImage.ts

```diff
@@ -13,6 +13,7 @@ export default function resetImage(editor: IEditor, image: HTMLImageElement) {
         image.style.maxWidth = '100%';
         image.removeAttribute('width');
         image.removeAttribute('height');
+        image.setAttribute('wasReset', 'true');
```

Comment:
If you use ImageEditInfo to save the result, you don't need to set anything here because next line (line 17) will delete the image edit info object so that image resized flag is also deleted.